### DISCUSSION TITLE
Google的新型可以对接吗

### DISCUSSION THREAD

**Discussion by jy00566722** [yangjian102621/chatgpt-plus#85] [常规讨论]
虽然google赶了一个晚集，但TA的东西肯定不会差啊，希望对接起来。不比国内那些瘪三货强多了啊。

**Comment by yangjian102621** [comment]
感谢反馈，Google 的 Gemini 模型有接入计划的，后面因为事情比较多就忘了。

**Reply by jy00566722** [reply]
早上好。
看到大佬回复马上来回一下：
我看到一些大的第三方token的站，开始出售mj画图了，他们用的是这个接入的：
https://github.com/novicezk/midjourney-proxy/
这个还有plus版本。

对接这个的话，就跟最初单独部署MJ机器人，再用chatgpt-plus去对接一样。但现在chatgpt-plus已经把mj机器人集成了，再去改也麻烦。

**Reply by jy00566722** [reply]
所以，按昨天群里的说法，是不是可以做成函数调用的形式,去调用他们的机器人。

**Reply by jy00566722** [reply]
不过mj的操作参数挺多，做成函数形式貌似不合适。

**Reply by yangjian102621** [reply]
这个没必要了，MJ 绘图我们最早就集成到了对话中的，只是后来 MJ 绘图做成单独的功能就把这个功能移除掉了，换成 DALLE3。

以前就是这种方式调用的，还支持放大（U）和变换（V）。

![image](https://github.com/yangjian102621/chatgpt-plus/assets/4746061/cd86993e-8ad5-4c29-b1b5-1761555309e4)

**Reply by jy00566722** [reply]
那现在去对接那个proxy,用什么形式比较合适，函数还是在现有mj画图那包一层。毕竟他们卖token的用这个，那我们用的直接买token最方便。

**Comment by yangjian102621** [comment]
> 那现在去对接那个proxy,用什么形式比较合适，函数还是在现有mj画图那包一层。毕竟他们卖token的用这个，那我们用的直接买token最方便。

你想把 MJ 接入函数其实也很简单，之前的代码还没有删除，参数多不是问题，反正 MJ 的参数都可以放入提示词。
